src/turret.cpp

```diff
@@ -78,6 +78,20 @@ turret_data vehicle::turret_query( const tripoint &pos ) const
     return const_cast<vehicle *>( this )->turret_query( pos );
 }
 
+bool vehicle::is_manual_turret( vehicle_part &pt ) const
+{
+    const auto &parts = parts_at_relative( pt.mount, false );
+    return std::any_of( parts.begin(), parts.end(),
+    [this]( int elem ) {
+        return part_info( elem ).has_flag( "MANUAL" );
+    } );
+}
+
+bool vehicle::is_manual_turret( const vehicle_part &pt ) const
+{
+    return const_cast<vehicle *>( this )->is_manual_turret( const_cast<vehicle_part &>( pt ) );
+}
```

Comment:
```suggestion
bool vehicle::is_manual_turret( const vehicle_part &pt ) const
{
    const auto &parts = parts_at_relative( pt.mount, false );
    return std::any_of( parts.begin(), parts.end(),
    [this]( int elem ) {
        return part_info( elem ).has_flag( "MANUAL" );
    } );
}
```

do we actaully need non const param version here? wouldn't it be possible to pass in both const and non-const vehicle_part to const param?

Reply:
Maybe, I wasn't sure. Will test in a sec.

Reply:
Aight, compiles once the vehicle.h version of the definition is also deleted so.